cmd/ctr/run_unix.go

```diff
@@ -222,6 +222,14 @@ func spec(id string, config *ocispec.ImageConfig, context *cli.Context, rootfs s
 			Type: "network",
 		})
 	}
+	for _, mount := range context.StringSlice("mount") {
```

Comment:
Seems like we should clarify that these mounts are being performed in the user namespace.